src/Symfony/Component/Security/Core/User/ChainUserProvider.php

```diff
@@ -91,7 +91,7 @@ public function refreshUser(UserInterface $user)
             $e->setUsername($user->getUsername());
             throw $e;
         } else {
-            throw new UnsupportedUserException(sprintf('The account "%s" is not supported.', \get_class($user)));
+            throw new UnsupportedUserException(sprintf('There is no user provider for user "%s.%s', \get_class($user), count($this->providers) > 0 ? ' Are you sure supportsClass() of your user provider returns true for this classname?' : ''));
```

Comment:
missing double quote after `%s`
+
`Shouldn't the "supportsClass()" method of your user provider returns true for this classname?`
same below

Reply:
👍 at this stage we can safely assume that a broken  `supportsClass` implementation is the cause

Reply:
👍 at this stage we can safely assume that a broken supportsClass implementation is the cause